lib/mayaUsd/ufe/UsdObject3d.cpp

```diff
@@ -75,7 +75,9 @@ Ufe::BBox3d UsdObject3d::boundingBox() const
     // as the time.
 
     auto bbox = UsdGeomImageable(fPrim).ComputeUntransformedBound(
-        getTime(sceneItem()->path()), UsdGeomTokens->default_);
+        getTime(sceneItem()->path()), UsdGeomTokens->default_,
+        UsdGeomTokens->guide, UsdGeomTokens->proxy, UsdGeomTokens->render
```

Comment:
On the proxy shape, we have toggles that control what will be rendered. I wonder if we shouldn't be respecting it here because purpose helps you manage complexity and including everything when computing bounding box may be simply too expensive.